packages/driver/src/multi-domain/cypress.ts

```diff
@@ -73,6 +76,18 @@ const setup = () => {
 
   specBridgeCommunicator.initialize(window)
 
+  // TODO Should state syncing be built into cy.state instead of being explicitly called?
+  specBridgeCommunicator.on('sync:state', async (state) => {
+    cy.state(state)
+  })
+
+  // Listen for window load events from the primary window to resolve page loads
+  specBridgeCommunicator.on('window:load', ({ url }) => {
+    cy.isStable(true, 'load')
```

Comment:
This one is kinda odd, without setting stable to true, commands aren't cleaned up properly. This can manifest in a couple of other ways too, failures or hot reloading. It deserves some more looking in to.

Reply:
Do we need a TODO for this?